tvbwidgets/core/pse/parameters.py

```diff
@@ -312,21 +319,21 @@ def compute_metrics(sim, metrics):
 
     for metric in metrics:
         if metric == "GlobalVariance":
-            resulted_metric = (GlobalVariance(sim.monitors[0].period))
+            resulted_metric = GlobalVariance(sim.monitors[0].period)
         elif metric == "KuramotoIndex":
-            resulted_metric = (KuramotoIndex(sim.monitors[0].period))
-        elif metric == "ProxyMetastabilitySynchrony Metastability":
-            resulted_metric = (ProxyMetastabilitySynchrony("Metastability", sim.monitors[0].period))
-        elif metric == "ProxyMetastabilitySynchrony Synchrony":
-            resulted_metric = (ProxyMetastabilitySynchrony("Synchrony", sim.monitors[0].period))
+            resulted_metric = KuramotoIndex(sim.monitors[0].period)
+        elif metric == "ProxyMetastabilitySynchrony-Metastability":
+            resulted_metric = ProxyMetastabilitySynchrony("Metastability", sim.monitors[0].period)
+        elif metric == "ProxyMetastabilitySynchrony-Synchrony":
+            resulted_metric = ProxyMetastabilitySynchrony("Synchrony", sim.monitors[0].period)
         else:
-            resulted_metric = (VarianceNodeVariance(sim.monitors[0].period))
+            resulted_metric = VarianceNodeVariance(sim.monitors[0].period)
         computed_metrics.append(resulted_metric)
 
     return computed_metrics
 
 
-def launch_local_param(simulator, param1, param2, x_values, y_values, metrics, file_name):
+def launch_local_param(param1, param2, x_values, y_values, metrics, file_name):
```

Comment:
I think we can put this parameter "simulator" back, and in the __main__ of this module, temporarily give it (for the HPC run) until we have that serialised properly